web/docs/language/operator-modifiers.md

```diff
@@ -8,7 +8,7 @@ Operator modifiers are keywords that may occur before an operator.
 
 ## Scheduled Executions
 
-The special keyword `every` enables scheduled execution of a source operator.
+The special keyword `every` enables scheduled execution of an operator.
```

Comment:
@mavam with this change we should think about putting operator modifiers into a more prominent place. My current thinking is that we may just want to document them as operators and stop distinguishing between operators and operator modifiers in the documentation in the first place.

It's a similar situation to void-to-void operators in that operator modifiers are hard to categorize as either source, transformation or sink, which our current docs sidebar layout enforces too strictly.